onboard/catkin_ws/src/acoustics/scripts/data_sim.py

```diff
@@ -8,28 +8,28 @@
 class DataGenerator:
     VS = 1511.5  # velocity of sound
 
-    def __init__(self, sampling_freq, pinger_freq, hydrophone, pinger_loc, publish):
+    def __init__(self, sampling_freq, pinger_freq, hydrophone_set, pinger_loc, publish):
         self.sf = sampling_freq
         self.pinger_frequency = pinger_freq
         self.publish = publish
         self.samples = int(math.floor(2.048 * 2 * self.sf))
         self.pinger_loc = pinger_loc
-        self.hydrophone = hydrophone
-        if self.hydrophone == 'cheap':
+        self.hydrophone_set = hydrophone_set
+        if self.hydrophone_set == 1:
             space = .3
             self.hp = [[0, 0, 0],
                        [space, 0, 0],
                        [0, space, 0],
                        [0, 0, space]]
-        elif self.hydrophone == 'expensive':
+        elif self.hydrophone_set == 2:
```

Comment:
This should use an enum (create two constants, one representing cheap and one expensive) defined in the message and compare to it there. I know I missed this in the saleae action, so change it there as well. Also, when you call the function, in the wrapper, you should use this enum instead of using the number 1 or 2